pkg/cluster/internal/kubeconfig/internal/kubeconfig/remove.go

```diff
@@ -48,6 +49,7 @@ func RemoveKIND(kindClusterName string, explicitPath string) error {
 				if err := write(existing, configPath); err != nil {
 					return err
 				}
+				logger.V(0).Infof("Removed context %q from kubeconfig", kindClusterName)
```

Comment:
We should note this and log it later, we should not log while holding locks, the logging is comparatively much slower than the other operations.

Reply:
How should I go about that?

Reply:
for what we're logging here currently we could toggle a boolean and then log based on that boolean after the logs are released